### PR TITLE
Inital test cases

### PR DESCRIPTION
**Pull request by hayatmahedavi**
Will be adding more test cases
currently just covering the happy path

### CODE REVIEW

File: www2/server/api/pricing/pricing.spec.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+var should = require('should');
```
Comment (jpilet) [inline www2/server/api/pricing/pricing.spec.js]: you should use const everywhere, not var

File: www2/server/api/pricing/pricing.spec.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            countries.errorMessage = err;
+            done(countries)
+        }
+    });
+
+    it('subscribe user to a plan', function (done) {
+        request(app)
+            .post('/api/pricing/subscribe/' + boatid)
+            .set('Authorization', 'Bearer ' + token)
+            // the name does not match, anemobox is missing. The test should fail.
```
Comment (jpilet) [inline www2/server/api/pricing/pricing.spec.js]: I do not understand this comment in this context...

File: www2/server/api/pricing/pricing.spec.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            .send({
+                stripeSource = "src_1FGs7eKcpwNgG6BNWXw6vaNf",
```
Comment (jpilet) [inline www2/server/api/pricing/pricing.spec.js]: Can you comment on how you got this string?

File: www2/server/api/pricing/pricing.spec.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                res.body.should.be.instanceof(Array);
+                done();
+            });
+    });
+
+
+    it('get prorations for a subscription', function (done) {
+        var subId = 'sub_FTtzxve542kq19';
+        request(app)
+            .get('/api/pricing/getProrations/' + subId)
```
Comment (jpilet) [inline www2/server/api/pricing/pricing.spec.js]: I would prefer if sending the auth token was mandatory to perform getProrations. But I can live with it as it is.

File: www2/server/api/pricing/pricing.spec.js (hunk 1 above)
Comment (hayatmahedavi) [reply www2/server/api/pricing/pricing.spec.js]: Done

File: www2/server/api/pricing/pricing.spec.js (hunk 2 above)
Comment (hayatmahedavi) [reply www2/server/api/pricing/pricing.spec.js]: It was a copy-paste mistake, removed the unwanted comment

**Pull request merged**